tiledb/sm/query/readers/result_tile.h

```diff
@@ -903,8 +950,14 @@ class GlobalOrderResultTile : public ResultTileWithBitmap<BitmapType> {
    */
   std::vector<BitmapType> extra_bitmap_;
 
-  /** Are duplicates allowed. */
-  bool dups_;
+  /**
+   * Delete condition index that deleted a cell. Used for consolidation with
+   * delete metadata.
+   */
+  std::vector<QueryCondition*> delete_condition_ptr_;
```

Comment:
Can we rename this to just `delete_conditions_` ? It's not recommended to add the type in the name in general, and I don't think it helps here, so I'd suggest removing those `ptr` names from this class in general